### DISCUSSION TITLE
Stream options no longer appear.

### DISCUSSION THREAD

**Discussion by luisboisvert** [maltejur/discord-screenaudio#231] [Q&A]
I'm on a Steam Deck using the flatpack version from Discover. I'm on 1.9.2 stable 279382 (5c40119).

Today when I start a stream, the usual audio/video options pop-up does not appear. I don't get the opportunity to disable my video or select an audio source, it just starts a stream showing my desktop.

**Comment by alteiaalteia** [comment]
Same one. I even try to download again, but it still the same
